dashboards/grafana-dashboard-assisted-installer-agent-api.configmap.yaml

```diff
@@ -71,7 +78,8 @@ data:
           "steppedLine": false,
           "targets": [
             {
-              "expr": "sum(rate(http_request_duration_seconds_count{endpoint=\"assisted-svc\", handler=\"/api/assisted-install/v1/clusters/{cluster_id}/hosts\", method=\"POST\"}[5m])) by (code)",
+              "exemplar": true,
+              "expr": "sum(rate(http_request_duration_seconds_count{endpoint=\"assisted-svc\", handler=\"/api/assisted-install/v2/infra-envs/{infra_env_id}/hosts\", method=\"POST\"}[5m])) by (code)",
```

Comment:
Probably out of scope for this change, but it would be best not to use hardcoded time interval as viewing this data for larger timeframes is not easily readable and it will take a toll on prometheus resources.